### ISSUE TITLE
Rename module Core to Svg.Core

### ISSUE THREAD

**Issue by epost** [kwohlfahrt/purescript-halogen-svg#9]
The current name looks like it was an oversight, is that right?

**Comment by kwohlfahrt** [comment]
Yes, I never noticed because I only import elements/attributes in my project. Will fix it later today.

**Comment by epost** [comment]
@kwohlfahrt Cool! Thanks. We also have a bunch of additions that I hope to clean up and contribute soon.
